apm-agent-core/src/main/java/co/elastic/apm/agent/configuration/CoreConfiguration.java

```diff
@@ -238,6 +240,33 @@ public static String getAllInstrumentationGroupNames() {
             WildcardMatcher.valueOf("(?-i)org.wildfly.security*")
         ));
 
+    private final ConfigurationOption<List<MethodMatcher>> traceMethods = ConfigurationOption
+        .builder(new ListValueConverter<>(MethodMatcherValueConverter.INSTANCE), List.class)
+        .key("trace_methods")
+        .configurationCategory(CORE_CATEGORY)
+        .description("A list of methods for with to create a transaction or span.\n" +
+            "\n" +
+            "The syntax is `modifier fully.qualified.class.Name#methodName(fully.qualified.parameter.Type)`.\n" +
+            "You can use wildcards for the class name, the method name and the parameter types.\n" +
+            "The `*` wildcard matches zero or more characters.\n" +
+            "Specifying the parameter types is optional.\n" +
+            "The `modifier` can be omitted or one of `public`, `protected`, `private` or `*`.\n" +
+            "\n" +
+            "A few examples:\n" +
+            "\n" +
+            " - `org.example.MyClass#myMethod`\n" +
+            " - `org.example.MyClass#myMethod()`\n" +
+            " - `org.example.MyClass#myMethod(java.lang.String)`\n" +
+            " - `org.example.MyClass#myMe*od(java.lang.String, int)`\n" +
+            " - `private org.example.MyClass#myMe*od(java.lang.String, *)`\n" +
+            " - `* org.example.MyClas*#myMe*od(*.String, int[])`\n" +
+            " - `public org.example.services.*.*Service#*`\n" +
+            "\n" +
+            "NOTE: Only use wildcards if necessary.\n" +
+            "The more methods you match to more overhead will be caused by the agent.\n" +
+            "Also note that there is a maximum amount of spans per transaction (see <<config-transaction-max-spans, `transaction_max_spans`>>).")
+        .buildWithDefault(Collections.singletonList(MethodMatcher.of("org.springframework.web.servlet.DispatcherServlet#render(*, *, *)")));
```

Comment:
Remove default, that was just for testing purposes. When actually instrumenting `DispatcherServlet#render`, we probably want to set a custom type. Maybe template or render.